### PR TITLE
Run tests/rails from root folder

### PR DESCRIPTION
**Pull request by tabfugnic**
By doing this it will provide a more direct output to what is
happening in the tests. It also means that you can run the app from
the root folder without having to navigate to the dummy app

* Feature specs run from root
* Single feature spec route
* Do not run rake task twice
* Provide simple Procfile for running rails server
* stub out rails and rspec
* bin/setup is much more simple now
* Upgrade dependencies in gemspec to match what is needed for
  application
* Silence output from puma when running tests

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -1,14 +1,18 @@
-#!/bin/sh
+#!/usr/bin/env sh
```
Comment (mike-burns) [inline bin/setup:1]: Is this for Solaris 8?

**Review by mike-burns — APPROVED** [review]
Nice!

File: bin/setup (hunk 1 above)
Comment (tabfugnic) [reply bin/setup:1]: Yes? I don't actually know why this got changed, it's possible I copied someone else's setup script and this happened.

**Comment by mike-burns** [comment]
LGTM, but I'd rather `/bin/sh` than looking it up via env(1).

**Pull request merged**